st2actions/st2actions/notifier/notifier.py

```diff
@@ -112,12 +111,12 @@ def _post_notify_triggers(self, liveaction=None, execution_id=None):
                 liveaction=liveaction, execution_id=execution_id,
                 notify_subsection=notify.on_complete,
                 default_message_suffix='completed.')
-        if liveaction.status == LIVEACTION_STATUS_SUCCEEDED and notify.on_success:
+        if liveaction.status == action_constants.LIVEACTION_STATUS_SUCCEEDED and notify.on_success:
             self._post_notify_subsection_triggers(
                 liveaction=liveaction, execution_id=execution_id,
                 notify_subsection=notify.on_success,
                 default_message_suffix='succeeded.')
-        if liveaction.status == LIVEACTION_STATUS_FAILED and notify.on_failure:
+        if liveaction.status == action_constants.LIVEACTION_STATUS_FAILED and notify.on_failure:
```

Comment:
Also, lol, I first thought you commented on my PR, but that's @m4dcoder's PR.

In any case, my branch / PR is WIP and should be ready on Monday (https://github.com/StackStorm/st2/compare/retry_policy_improvements).